### ISSUE TITLE
Enable KV cache PCC check for benchmark LLMs

### ISSUE THREAD

**Issue by vkovacevicTT** [tenstorrent/tt-xla#4783]
Currently in benchmarks PCC is calculated for prefill and first decode logits (for User 0).

Next step should be enabling KV cache PCC check.
